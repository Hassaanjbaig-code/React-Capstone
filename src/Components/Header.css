.head {
  width: 100%;
  height: 4rem;
  display: flex;
  align-items: center;
  justify-content: space-between;
}

.Icon {
  font-size: 1.7rem;
}

.head h3 a {
  text-decoration: none;
  color: #69123c;
}

.Setting {
  margin-right: 12px;
}
