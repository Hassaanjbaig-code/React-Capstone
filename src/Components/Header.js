import React from 'react';
import { NavLink } from 'react-router-dom';
import { FaAngleLeft } from 'react-icons/fa';
import './Header.css';

const Header = () => {

  return (
    <div className="head">
      <NavLink to='/'>
        <FaAngleLeft  className="Icon"/>
      </NavLink>
        <h3>US Technology Company</h3>
    </div>
  )
}

export default Header
