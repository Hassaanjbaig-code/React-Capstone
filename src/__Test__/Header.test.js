import React from 'react';
import '@testing-library/jest-dom';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import Header from '../Components/Header';
import store from '../redux/Store';

describe('test/navbar', () => {
  test('should first', () => {
    const tree = renderer.create(
      <React.StrictMode>
        <Provider store={store}>
          <BrowserRouter>
            <Header />
          </BrowserRouter>
        </Provider>
      </React.StrictMode>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
