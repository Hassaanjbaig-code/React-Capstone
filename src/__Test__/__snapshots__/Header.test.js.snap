// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test/navbar should first 1`] = `
<div
  className="head"
>
  <a
    aria-current="page"
    className="active"
    href="/"
    onClick={[Function]}
  >
    <svg
      className="Icon"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 256 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
      />
    </svg>
  </a>
  <h3>
    <a
      href="https://www.indmoney.com/us-stocks/top-tech-stocks"
    >
      US Technology Company
    </a>
  </h3>
</div>
`;
